src/model/advertisement.ts

```diff
@@ -19,7 +21,7 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'top-above-nav',
```

Comment:
For top-above-nav, there's also [Commercial.scala](https://github.com/guardian/frontend/blob/b6cb79e726432767b68ff22ddfc3e5fd3392a1c2/common/app/views/support/Commercial.scala#L128) which seems to have sizing code in it - not sure which is right 🤷‍♂

Reply:
Done